File: Selenium/Activities/Activity1.py
# Import required packages from selenium

from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager

# Set up the Firefox Driver using WebDriverManger
service = FirefoxService(GeckoDriverManager().install())

# Start the Driver
driver=webdriver.Firefox(service=service)

# Launch the URL and get the title
driver.get("https://v1.training-support.net/")
home_title=driver.title
print ("Title of the Home Page is {}".format(home_title))

# get the aboutus link, click on it and get the aboutus page title
aboutus_link=driver.find_element(By.ID, "about-link")
aboutus_link.click()
aboutus_title=driver.title
print ("Title of the aboutus Page is {}".format(aboutus_title))

# CLose the Browser
driver.close()

File: Selenium/Activities/Activity1.java
package activities;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity1 {
    public static void main(String[] args) {
        // Setup Firefox Driver
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();

        // Open the URL and get the title
        driver.get("https://v1.training-support.net");
        String title = driver.getTitle();
        System.out.println("Title of the Home Page is : " + title);

        // Get the aboutus Link and click on it and print the title
        WebElement aboutUsLink= driver.findElement(By.id("about-link"));
        aboutUsLink.click();
        String aboutUsTitle=driver.getTitle();
        System.out.println("Title of the AboutUs Page is : " + aboutUsTitle);

        // Close the Browser
        driver.close();

    }
}
